Kinvey/Kinvey/Error.swift

```diff
@@ -89,6 +89,14 @@ public enum Error: Swift.Error, CustomStringConvertible, CustomDebugStringConver
         }
     }
     
+    public var errorDescription: String? {
+        return description
+    }
+    
+    public var failureReason: String? {
+        return description
```

Comment:
Hmm, perhaps I was mistaken: https://developer.apple.com/library/content/documentation/Cocoa/Conceptual/ErrorHandlingCocoa/ErrorObjectsDomains/ErrorObjectsDomains.html#//apple_ref/doc/uid/TP40001806-CH202-SW4

> A brief sentence that explains the reason why the error occurred. It is typically part of the error description.

Also:

> An example can help to clarify the relationship between error description and failure reason. An error object has an error description of “File could not be saved because the disk is full.” The accompanying failure reason is “The disk is full.”